kr/kr_darwin.go

```diff
@@ -110,7 +110,7 @@ func copyEnvToLaunchctl(varName string) {
 	_, _ = runCommandTmuxFriendly("launchctl", "setenv", varName, os.Getenv(varName))
 }
 
-func restartCommand(c *cli.Context) (err error) {
+func restartCommandOptions(c *cli.Context, showPrints bool) (err error) {
```

Comment:
with this refactor the analytics will print even for a transfer command. can we rename `showPrints` and only have send analytics when it is true?

Reply:
`showPrints` -> `isUserInitiated` 👍